tests/bsim/bluetooth/audio/src/cap_acceptor_test.c

```diff
@@ -877,6 +989,93 @@ static void test_cap_acceptor_broadcast(void)
 	PASS("CAP acceptor broadcast passed\n");
 }
 
+static void test_cap_acceptor_broadcast_reception(void)
+{
+	static struct bt_bap_stream *bap_streams[ARRAY_SIZE(broadcast_sink_streams)];
+	size_t stream_count;
+	int err;
+
+	init();
+
+	WAIT_FOR_FLAG(flag_pa_request);
+
+	err = pa_sync_create();
+	if (err != 0) {
+		FAIL("Could not create Broadcast PA sync: %d\n", err);
+		return;
+	}
+
+	printk("Waiting for PA sync\n");
+	WAIT_FOR_FLAG(flag_pa_synced);
```

Comment:
This test function was copied from the test_cap_acceptor_broadcast test, so that one should than be changed as well. So I propose to do it in a separate PR as a cleanup job

Reply:
> This test function was copied from the test_cap_acceptor_broadcast test, so that one should than be changed as well. So I propose to do it in a separate PR as a cleanup job

I disagree. There's no point in adding new functions to just fix them later. Makes more sense to add them "correctly" the first time

Reply:
The function pa_sync_create already existed in this module. Cleanup implies modifying existing tests, and will be done in a separate PR